src/main/java/graphql/execution2/FetchedValueAnalysis.java

```diff
@@ -135,21 +96,19 @@ public static Builder newFetchedValueAnalysis(FetchedValueType valueType) {
         return new Builder().valueType(valueType);
     }
 
+    public static Builder newBuilder(FetchedValueAnalysis existing) {
```

Comment:
Naming

All our builders tend to be newFoo not newBuilder.

This should be `newAnalysis` or `newFetchedValueAnalysis` (I would go with the former)